unstable/wlr-output-manager-v1.xml

```diff
@@ -0,0 +1,350 @@
+      <description summary="human-readable description of this output">
+        Many compositors can produce human-readable descriptions of their
+        outputs.  The client may wish to know this description as well, to
+        communicate the user for various purposes.
+
+        The description is a UTF-8 string with no convention defined for its
+        contents. Examples might include 'Foocorp 11" Display' or 'Virtual X11
+        output via :1'.
+
+        This event is only sent once per zwlr_output_device_v1, and the
+        name does not change over the lifetime of the zwlr_output_device_v1.
+      </description>
+      <arg name="description" type="string" summary="output description"/>
+    </event>
+
+    <event name="done">
+      <description summary="sent all information about output">
+        This event is sent after all other properties have been
+        sent after binding to the output object and after any
+        other property changes done after that. This allows
+        changes to the output properties to be seen as
+        atomic, even if they happen via multiple events.
+      </description>
+    </event>
+
+    <event name="disconnected">
+      <description summary="">
+        The disconnected event indicates that the output device is no longer
+        connected to the compositor. The server will no longer send events to
+        this object.
+
+        If the client tries to configure any of the output's properties, it will
+        fail.
+
+        The client should destroy the object after it receives this event.
+      </description>
+    </event>
+
+    <request name="destroy" type="destructor">
+      <description summary="destroy the output device object">
+        Using this request a client can tell the server that it is not going to
+        use the output device object anymore.
+      </description>
+    </request>
+
+  </interface>
+
+  <interface name="zwlr_output_manager_v1" version="1">
+    <description summary="manage output device properties">
+      Allows the client to change the properties of an output device. The
+      client binds the output manager globally, then creates a configuration
+      object.
+    </description>
+    <request name="create_configuration">
+      <description summary="create a new output configuration object">
+        Create a new output configuration object.
+      </description>
+      <arg name="id" type="new_id" interface="zwlr_output_device_v1"/>
+    </request>
+
+    <request name="destroy" type="destructor">
+      <description summary="destroy the output manager">
+        Using this request a client can tell the server that it is not going
+        to use the manager object anymore. Any objects created by the manager
+        will remain valid, until their appropriate destroy request has been
+        called.
+      </description>
+    </request>
+  </interface>
+
+  <interface name="zwlr_output_configuration_v1" version="1">
```

Comment:
How do you set the scale?

Reply:
No, I mean there's no `set_scale` request in this interface.

Reply:
> Added size. Similar to xdg-output's logical_size event. Allows for fractional scaling.

It is computable from this (provided I understood the intended usecase correctly)
But I wonder if it's a good idea to have this in the long run

Reply:
This allows to read the scale, not to set it.

Reply:
My bad, must have forgotten to add the set_size event. I don't think we should drop fractional scaling in this protocol, since it is used for configuration instead of rendering. I used this on X on my old laptop to scale from 1600x900 (built in screen resolution) to 1920x1080 for certain external displays when giving presentations.

Reply:
I'm not sure a `set_size` request would work well: what happens if the compositor doesn't support fractional scaling? What happens if the x-axis ratio is different from the y-axis ratio?

Reply:
The compositor would reject the change in that case.